config/parameters.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  plus_plus.increased_messages:

```

Comment:
I Think would be nice if this message came from outside of the code, maybe on the environment file, i am not sure if it's possible defined array inside .env, i think if we need to add another message we need to add on this part and make another deploy, if this came from .env you don't need do neither of that, so, what do you think?  this it's only an alternative open to know your opinion about it , by the moment i approve this in the meanwhile

Reply:
Nice suggestion, I think it is better to have a PR to review the messages and avoid any kind of error or make suggestions about them.

Also we didn't require the access to the server to review or change those messages 
(the deployment is another topic jeje).

But definitively it is a good idea, what do you suggest @eruizdechavez ?

Reply:
I like the idea of keeping the messages outside the code. My suggestion would be to use what we already have: SQLite.

￼Why not add a new table and￼ it's ￼default data in a migration?

Reply:
By the way, it doesn't need to be a messages table, it can be a new settings table with 3 rows, one for each setting.

Reply:
To use a database also sounds good if the intention is to have an interface to edit this settings, otherwise, change the messages can be hard, in special if we are going to store this message in a single db column as a JSON or some like that.

I think that the easier way to mantain this messages right now its the yaml file. What do you think if we discuss what is the best approach to configure the messages, in the meantime we can handle the messages in this way?